sdk/storage/Azure.Storage.Blobs/tests/BlobQuickQueryTests.cs

```diff
@@ -126,14 +126,14 @@ public async Task QueryAsync_MultipleDataRecords()
         }
 
         [Test]
-        [Ignore("Don't want to record 120 MB of data.")]
+        [Ignore("Don't want to record 250 MB of data.")]
```

Comment:
Should this be live test ?

Reply:
I think it will timeout and/or be flaky.

Reply:
We can give it a try and see what happens.

Reply:
We should never use `[Ignore]` without a work item.  If we're not planning to make the test stable someday, we might as well delete it now.

Reply:
This test use useful for local testing, to make sure we didn't break the large query cases.

Reply:
Maybe we should spin up additional test project that's run separately from nightly tests - i.e. before release and locally on demand. We've been struggling to find a good home for resource heavy tests like that for a while.
Also our libraries promise to handle large payloads well, so we should have an automated way to verify this aspect. If infra is a problem here let's invest in infra.